e2e/runner/run_cypress_ci.js

```diff
@@ -31,13 +35,34 @@ const startServer = async () => {
   await CypressBackend.start();
 };
 
-const runTests = async (testSuite) => {
-  printBold(`Running ${testSuite} Cypress Tests`);
-  await runCypress(testSuite, process.exit);
+const runTests = async (config, cliArguments = []) => {
+  const userOverrides = await parseArguments(cliArguments);
+  await runCypress({ ...config, ...userOverrides }, process.exit);
 };
 
-if (modeOrTestSuite === "start") {
+// Custom "modes"
+if (command === "start") {
   startServer();
-} else {
-  runTests(modeOrTestSuite);
+}
+
+if (command === "snapshot") {
+  runTests({ configFile: "e2e/support/cypress-snapshots.config.js" });
+}
+
+// Metabase component or e2e tests
+if (command === "component") {
+  runTests({
+    configFile: "e2e/support/cypress-embedding-sdk-component-test.config.js",
+    testingType: "component",
+  });
+}
+
+if (command === "e2e") {
+  runTests({ configFile: "e2e/support/cypress.config.js" }, cliArguments);
+}
```

Comment:
We've come to the point where there's basically no difference between the cypress cli vs cypress node module invocation. All these can easily be called from the terminal with:

```
cypress run --config-file e2e/support/cypress-snapshots.config.js

cypress run --config-file e2e/support/cypress-embedding-sdk-component-test.config.js

cypress run --config-file e2e/support/cypress.config.js
```